velox/tool/trace/TraceReplayRunner.cpp

```diff
@@ -108,6 +108,10 @@ DEFINE_string(
     function_prefix,
     "",
     "Prefix for the scalar and aggregate functions.");
+DEFINE_string(
+    spill_directory,
+    "",
+    "Base directory for spilling. If not specified, a local temporary directory will be used.");
```

Comment:
Nit: "Base directory for spilling. Defaults to a local temporary directory if not specified."